src/client/App.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
-import Home from './containers/home/home.component';
+import LoginPage from './containers/login-page/login-page';
+import ForgotPassword from './containers/forgot-password/forgot-password';
+import Dashboard from './containers/dashboard-page/dashboard-page';
+import Firebase, { FirebaseContext } from './firebase/index';
 
 function App() {
   return (
     <Router>
       <Switch>
+        <Route exact path="/forgot-password">
+          <ForgotPassword />
+        </Route>
+        <Route exact path="/dashboard">
+          <Dashboard />
+        </Route>
         <Route exact path="/">
-          <Home />
+          <FirebaseContext.Provider value={Firebase.init()}>
+            <LoginPage />
+          </FirebaseContext.Provider>
         </Route>
```

Comment:
I think I would wrap the entire app in the context provider as you might want to be able to display state about whether you are signed in or not in some of the pages even though they don't require authentication 👍

Reply:
Done